api/HTMLMediaElement.json

```diff
@@ -2871,17 +2871,20 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLMediaElement/srcObject",
           "spec_url": "https://html.spec.whatwg.org/multipage/media.html#dom-media-srcobject-dev",
           "support": {
-            "chrome": {
-              "version_added": "52",
-              "partial_implementation": true,
-              "notes": "Only supports <code>MediaStream</code> objects (see <a href='https://crbug.com/506273'>bug 506273</a>)."
-            },
+            "chrome": [
+              {
+                "version_added": "108",
+                "partial_implementation": true,
+                "notes": "Support added for <code>MediaSource</code> objects transferred from dedicated workers via <code>MediaSource.handle</code> (see <a href='https://crbug.com/878133'>bug 878133</a>)."
```

Comment:
Probably best to be precise here: ...Support added for <code>MediaSource**Handle**</code> objects transferred from dedicated workers where they were obtained from <code>MediaSource.handle</code>...

Reply:
Good call, updated!